### ISSUE TITLE
train loss NAN

### ISSUE THREAD

**Issue by racso10** [spatialsr/DeepLightFieldSSR#4]
Hello henrywfyeung,
I try to run the 4D training code with the command `train_Spatial_SR(2,10,1)`. But the loss quickly increases until `NAN` or `Inf`, which means that the model will not converge.
We use the Stanford training dataset which contains 130 LFs, as stated in the README.
Could you tell me why this is happening and how should we solve this problem?

Thanks a lot
